inc/uf2.h

```diff
@@ -69,6 +71,7 @@
 #define USE_MSC_CHECKS 0   // check validity of MSC commands; 460 bytes
 #define USE_CDC_TERMINAL 0 // enable ASCII mode on CDC loop (not used by BOSSA); 228 bytes
 #define USE_DBG_MSC 0      // output debug info about MSC
+#define USE_BINARY_FILES 0 // enable Arduino bin-files in MSC; 296 bytes
```

Comment:
.bin files don't describe a load offset, so where should they go?

Reply:
Sorry, I was using an old copy from [here](https://github.com/awatterott/uf2-samd21).